MTM-Backend/src/routes/v1/cashDonation/cashDonation.service.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Why are you not using type annotations here? Let's try bringing them back and add type annotations to stick to type rules